src/View/Helper/FormHelper.php

```diff
@@ -191,6 +191,14 @@ public function input($fieldName, array $options = [])
         $options = $this->_parseOptions($fieldName, $options);
         $reset = $this->templates();
 
+        $templater = $this->templater();
+        if (!empty($options['templates'])) {
```

Comment:
Do you need isset()? Otherwise the key+value will stick around in the options array.